uk.co.compucorp.civicrm.tasksassignments/api/v3/Task.php

```diff
@@ -322,6 +322,7 @@ function civicrm_api3_task_delete($params) {
   }
   
   if (CRM_Activity_BAO_Activity::deleteActivity($params)) {
+    CRM_Tasksassignments_Reminder::sendReminder($params['id']);
```

Comment:
Thanks @Agares4 
If we did that and activity is not deleted because of some error, wouln't that cause issues maybe ?